pkg/statsd/handler_backend.go

```diff
@@ -43,10 +43,12 @@ func NewBackendHandler(backends []gostatsd.Backend, maxConcurrentEvents uint, nu
 
 	for i := 0; i < numWorkers; i++ {
 		workers[i] = &worker{
-			aggr:         af.Create(),
-			metricsQueue: make(chan *gostatsd.Metric, perWorkerBufferSize),
-			processChan:  make(chan *processCommand),
-			id:           i,
+			aggr: af.Create(),
+			// TODO: Reassess the defaults
+			metricsQueue:   make(chan []*gostatsd.Metric, perWorkerBufferSize),
+			metricMapQueue: make(chan *gostatsd.MetricMap, perWorkerBufferSize),
```

Comment:
This channel isn't monitored.  I'll fix it when I fix the broken metrics generally (one of the refactors stopped calling `RunMetrics`)